Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM python:3.7-slim
```

Comment:
If we want to add support for Eclipser and AFL, we will most likely need to use a different, heavier base image. I did a spike with https://github.com/SoftSec-KAIST/Eclipser-Artifact as a base image, but it uses Ubuntu 16.04 for its base image (which doesn't include python3.6 so it turned into a nightmare to try to get everything to work correctly)